src/executors/HybridSimulatorDDExecutor.cpp

```diff
@@ -0,0 +1,32 @@
+#include "executors/HybridSimulatorDDExecutor.hpp"
+
+#include "HybridSchrodingerFeynmanSimulator.hpp"
+
+json HybridSimulatorDDExecutor::execute(const SimulationTask& task) {
+  json       result;
+  auto const constructionStart = std::chrono::steady_clock::now();
+
+  auto qc = std::make_unique<qc::QuantumComputation>(task.getQc()->clone());
+  auto hybridSimulator = std::make_unique<HybridSchrodingerFeynmanSimulator<>>(
+      std::move(qc), ApproximationInfo{}, 23,
```

Comment:
Extract the magic seed number as a global constant somewhere so that it can be used everywhere and is just defined in a single place.